src/CattyTests/RotationStyleTests.swift

```diff
@@ -0,0 +1,78 @@
+/**
+ *  Copyright (C) 2010-2020 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+
+import XCTest
+
+@testable import Pocket_Code
+
+final class RotationStyleTests: XCTestCase {
```

Comment:
Please move to the `SpriteObject` folder.

Reply:
<img width="268" alt="Bildschirmfoto 2020-10-30 um 08 40 52" src="https://user-images.githubusercontent.com/56385054/97673211-aec73800-1a8b-11eb-86fe-da336c176a47.png">
It is?

Reply:
Only in Xcode, not in the file system.

Reply:
Done!